modules/ROOT/pages/plugins/premium/export.adoc

```diff
@@ -6,23 +6,24 @@
 :pluginname: Export
 :plugincode: export
 
-include::partial$misc/requires_5_5v.adoc[] +
+include::partial$misc/requires_5_5v.adoc[]
+
 include::partial$misc/premiumplugin.adoc[]
```

Comment:
Just confirming that the content in this include has been fixed elsewhere? Asking as it's rendering a link incorrectly atm:
![image](https://user-images.githubusercontent.com/1575550/159800590-8c56542c-3174-4a28-81e4-432361ad40e9.png)

Reply:
Yes it has been fixed in another PR